pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1258,7 +1258,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
+1 there is a lot of confusion here.
Opened #11712 to track documentation for adding back APIs as well as adding new APIs. 
@stephentoub, feel free to reassign that to the proper owner.

/cc @ericstj @weshaggard @Petermarcu